website/docs/language/functions/urldecode.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_label: urldecode
+description: The urldecode function applies URL decoding to a given string.
+---
+
+# `urldecode` Function
+
+`urldecode` targets encoded characters within a string. 
+
+The function is capable of decoding a comprehensive range of characters, 
+including those outside the ASCII range. Non-ASCII characters are first treated as UTF-8 bytes,
+followed by the application of percent decoding to each byte, 
+facilitating the accurate decoding of multibyte characters.
+
+
+## Implementation Details
+
+The decoding process includes a safety check against null bytes (`%00`),
+which are prohibited due to their potential to introduce security vulnerabilities.
+Upon encountering such sequences, the function halts and returns an error,
+signaling the presence of invalid input.
+
+## Examples
+
+```
+> urldecode("Hello+World%21")
+Hello World!
+> urldecode("%E2%98%83")
+☃
+> "foo%3Abar%40localhost%3Ffoo%3Dbar%26bar%3Dbaz"
```

Comment:
I believe that in this example, we forgot to include the `urldecode()` function.